xml/System.ServiceModel.Federation/WSTrustChannel.xml

```diff
@@ -1,602 +1,602 @@
+      <MemberType>Event</MemberType>
+      <Implements>
+        <InterfaceMember>E:System.ServiceModel.ICommunicationObject.Opening</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when the communication object first enters the opening state.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Federation.WSTrustChannel&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.State">
+      <MemberSignature Language="C#" Value="System.ServiceModel.CommunicationState System.ServiceModel.ICommunicationObject.State { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.ServiceModel.CommunicationState System.ServiceModel.ICommunicationObject.State" />
+      <MemberSignature Language="DocId" Value="P:System.ServiceModel.Federation.WSTrustChannel.System#ServiceModel#ICommunicationObject#State" />
+      <MemberSignature Language="VB.NET" Value=" ReadOnly Property State As CommunicationState Implements ICommunicationObject.State" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.ICommunicationObject.State : System.ServiceModel.CommunicationState" Usage="System.ServiceModel.ICommunicationObject.State" />
+      <MemberSignature Language="C++ CLI" Value="property System::ServiceModel::CommunicationState System::ServiceModel::ICommunicationObject::State { System::ServiceModel::CommunicationState get(); };" />
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.ICommunicationObject.State</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.ServiceModel.CommunicationState</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the current state of the communication-oriented object.</summary>
+        <value>The value of the <see cref="T:System.ServiceModel.CommunicationState" /> of the object.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Federation.WSTrustChannel&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Federation.WSTrustChannel"/> instance is cast to an <see cref="T:System.ServiceModel.ICommunicationObject"/> interface.</remarks>

```